### ISSUE TITLE
Reminder for author response?

### ISSUE THREAD

**Issue by maelle** [ropensci-org/buffy#72]
@mpadge (& others) do you think it'd make sense to have a reminder for the author response 12 days or so after both reviews are in?

It'd however require a way to detect the author response once it's been posted, as the editor might not change the label right away. So should the authors use something like `@ropensci-review-bot submit response <url-to-response>`? :thinking:

**Comment by mpadge** [comment]
Yes! And having a `submit response` would solve problems that would otherwise arise by requiring authors themselves to define their response. I think something like that would be perhaps the only workable way to implement this.

**Comment by maelle** [comment]
Thanks @mpadge! 

@xuanxu here's a new feature request. :pray: 

- Having a "submit response" responder for authors. `@ropensci-review-bot submit response <url to issue comment>`.
- 12 days after the label "4/review(s)-in-awaiting-changes" is applied, post a reminder comment tagging all authors (author1 but also author-others if there are some)

```
@github-handle please post your response with `@ropensci-review-bot submit response <url to issue comment>`.

Here's the author guide for response. https://devguide.ropensci.org/authors-guide.html
```

**Comment by xuanxu** [comment]
@maelle A couple of questions:
- When the author uses the "submit response" command what should happen?
- Who can use the command? only the `author1` or `author-others` too?

**Comment by maelle** [comment]
> Who can use the command? only the author1 or author-others too?

Both kinds of authors.

I'll update this thread again when I have a good answer for the first question.

**Comment by maelle** [comment]
> When the author uses the "submit response" command what should happen?

Adding an entry to the authors-response table.
id_no just a unique number 
response_date datetime at which the response is submitted
package link to packages table record
URL of response
